### PR TITLE
Docs/sydney 190 explorer

### PR DESCRIPTION
**Pull request by sydstoplight**

### CODE REVIEW

**Review by pamgoodrich — CHANGES_REQUESTED** [review]
Hi, Sydney. Nice job overall. I left some comments. I did not realize until Troy pointed it out that we have some of this documented here: 

https://meta.stoplight.io/docs/platform/ZG9jOjU4NzU5Ng-exploring-your-api-projects

I'll remove that topic as part of the TOC work, but can you please review it and verify that we have relevant content in your new topics? Thank you!

File: 2.-workspaces/m.groups.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ Groups allow you to organize projects in your sidebar and grant project access t
```
Comment (pamgoodrich) [inline 2.-workspaces/m.groups.md:9]: I am not sure why this file is in this branch. It looks like you may have created your branch from something other than main. Remember to always switch to main, pull, and then create your new branch. I will fix the merge conflict on this file after you finish your review updates.

File: explorer/overview.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+* Search and Filter
+* View Changelogs
+* Track Dependencies
+* Star Repositories
+* Try Out APIs
+
+To access Explorer within your workspace, select **Explore** on the left pane.
+
+![explore_navigation.png](../assets/images/explore_navigation.png)
+
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/overview.md:28]: Added a blur to all screenshots for logged in user and project names

File: explorer/overview.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+tags: [Explorer]
+---
+
+# Overview
+
+Poor visibility into available APIs lead to inconsistencies and duplications that suffer costs to your business. Explorer provides visibility by acting as a central repository for all design assets across your organization. Stoplight Explorer makes it easy to find, search, and manage API artifacts.
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/overview.md:7]: Complete

File: explorer/overview.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+After adding projects to your workspace, Explorer analyzes them and creates an indexed, searchable view of your projects.
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/overview.md:9]: Complete

File: explorer/overview.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+The assets discoverable in Explorer include:
+* API Descriptions
+* Models/Schemas
+* Documentation
+* Style guides
+* Design documents
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/overview.md:16]: Complete

File: explorer/overview.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+With API artifacts in Explorer, you can: 
+* Search and Filter
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/overview.md:19]: Complete

File: explorer/overview.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+* Track Dependencies
+* Star Repositories
+* Try Out APIs
+
+To access Explorer within your workspace, select **Explore** on the left pane.
+
+![explore_navigation.png](../assets/images/explore_navigation.png)
+
+The explorer lists all of your organization's assets. Clicking on individual assets allows you to explore them further.
+
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/overview.md:30]: Complete

File: explorer/search-and-filter.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+tags: [Explorer]
+---
+
+# Search and Filter Components
+
+Search for assets from your API project and filter assets based on their type using **Explorer**.
+
+### Use Search and Filter
+
+API assets compile quickly. With multiple APIs and projects, it becomes difficult to navigate API assets.
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/search-and-filter.md:11]: Complete

File: explorer/search-and-filter.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+Explorer provides visibility to help you navigate projects. **Search** finds specific assets or multiple assets with identical names. **Filter** provides options for browsing through similar assets. 
+
+1. Log in to your workspace, then select **Explore** on the left pane.
+2. Enter the asset name into the top pane next to the **Search** icon.
+
+![explore_navigation.png](../assets/images/explore_navigation.png)
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/search-and-filter.md:18]: Added new image

File: explorer/view-changelogs.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+# View Changelogs
+
+Use Explore to view a list of dated changes to any asset type in your API project.
+
+
+1. Log in to your workspace, and then select **Explore** on the left pane. Select your desired asset type.
+
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/view-changelogs.md:7]: Complete

File: explorer/search-and-filter.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+3. Select the **Filter** icon on the top pane.
+4. Select your desired asset type from the menu.
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/search-and-filter.md:21]: Complete

File: explorer/star-repositories.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
+# Star Repositories
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/star-repositories.md:1]: Complete

File: explorer/star-repositories.md [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+
+Navigate through hundreds of assets in your project by starring what you need.
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/star-repositories.md:3]: Complete

File: explorer/star-repositories.md [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+
+### Use Stars
+
+Stars in Explorer operate like bookmarks. You can star assets within your workspace, docs, endpoints, or models in order to easily revisit them.
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/star-repositories.md:7]: Complete

File: explorer/track-dependencies.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+
+<!-- theme: info -->
+
+> #### **Note:**
+>
+> This feature is included with the Professional Plan and higher.
+
+### Track Dependencies
+
+1. Log in to your workspace, and then select **Explore** on the left pane. Select your desired endpoint or model.
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/track-dependencies.md:25]: Complete

File: explorer/track-dependencies.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+tags: [Explorer]
+---
+
+# Track Dependencies
+
+Use Explorer to discover relationships in your data. View which APIs, endpoints, or models reference a particular model, and navigate outbound dependencies through a visual diagram.
+
+### Dependencies
+
+Overlapping references to models used in multiple endpoints often breaks changes before detection.
```
[the remark this answers is not in the document]
Comment (sydstoplight) [reply explorer/track-dependencies.md:11]: Complete

File: 2.-workspaces/m.groups.md (hunk 1 above)
Comment (sydstoplight) [reply 2.-workspaces/m.groups.md:9]: Complete

**Review by pamgoodrich — APPROVED** [review]

**Pull request merged**